RecallActivity.py

```diff
@@ -54,6 +44,7 @@ class RecallActivity(activity.Activity):
     def __init__(self, handle):
         """ Initialize the toolbars and the game board """
         super(RecallActivity, self).__init__(handle)
+        self.max_participants = 1
```

Comment:
I guess; we could remove this instance of `self.max_participants` if its possible to. then this PR would be perfect!
Attach files by dragging & dropping, selecting or pasting them.